### PR TITLE
Improve iOS share extension and syncing

### PR DESCRIPTION
**Pull request by jacksonh**
- Proper case-insensitive compare on the labels in the share extension
- Fix search in filter by labels view
- Use the ApplyLabels view when users want to search for labels
- Use custom fonts for text chips in the share extension
- Cut down on loading refreshes

### CODE REVIEW

File: apple/OmnivoreKit/Sources/App/AppExtensions/Share/ShareExtensionScene.swift [hunk 1]
```diff
@@ -1,8 +1,11 @@
 import SwiftUI
 import Utils
+import Views
 
 public extension PlatformViewController {
   static func makeShareExtensionController(extensionContext: NSExtensionContext?) -> PlatformViewController {
+    registerFonts()
```
Comment (satindar) [inline apple/OmnivoreKit/Sources/App/AppExtensions/Share/ShareExtensionScene.swift:7]: Oh good catch here!

File: apple/OmnivoreKit/Sources/Views/Resources/en.lproj/Localizable.strings [hunk 2]
```diff
@@ -10,8 +10,8 @@
 
 "labels.generic" = "Labels";
 "emails.generic" = "Emails";
-"subscriptions.generic" = "Text to Speech";
```
Comment (satindar) [inline apple/OmnivoreKit/Sources/Views/Resources/en.lproj/Localizable.strings:13]: This is what happens when I stop drinking real coffee 😬

**Review by satindar — APPROVED** [review]
Looks awesome! I'll do some more testing but will merge now.

File: apple/OmnivoreKit/Sources/Views/Resources/en.lproj/Localizable.strings (hunk 2 above)
Comment (jacksonh) [reply apple/OmnivoreKit/Sources/Views/Resources/en.lproj/Localizable.strings:13]: Hah, amazingly a user reported within an hour of testflight being published

**Pull request merged**